main.lua

```diff
@@ -1,7 +1,5 @@
 pico8={
 	fps=30,
-	frames=0,
-	pal_transparent={},
```

Comment:
What purpose does removing these lines here serve?

Reply:
Pal_transparent is set again some line below, frames is set in _load because otherwise is not set to 0 when you reload a cart without restarting picolove

Reply:
The problem of reloading was appearing in a second game where there’s an intro with some time() based animations and without changes it was skipping it at every reload because timing was not reset properly mainly because pico8.frames kept the value from before reset